examples/quick-start-net-core/tracetest.config.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
the rest of the examples are not using the initial `---`. I would keep the same standard just for consistency

Reply:
Also, the rest of the examples are using `tracetest-config.yaml` and `tracetest-provision.yaml` for the file names